namespace.go

```diff
@@ -52,6 +52,9 @@ const (
 	Version = "0.10.3"
 
 	rootUserAgent = "/golang-service-bus"
+
+	// default ServiceBus resource uri to authenticate with
+	serviceBusResourceURI = "https://servicebus.azure.net/"
```

Comment:
used for fallback to previous behavior in case the environment value is empty.
it is set for existing environment maps, but could be empty for a user provided map.